pinc/DifferenceEngineWrapper.inc

```diff
@@ -25,20 +25,28 @@ include_once($relPath."3rdparty/mediawiki/DairikiDiff.php");
 include_once($relPath."3rdparty/mediawiki/WordLevelDiff.php");
 include_once($relPath."3rdparty/mediawiki/DiffEngine.php");
 include_once($relPath."3rdparty/mediawiki/DiffFormatter.php");
-include_once($relPath."3rdparty/mediawiki/TableDiffFormatter.php");
```

Comment:
Line 21 (pref_options.inc include) needs to be moved into `pinc/DifferenceEngineWrapperTable.inc` as that's where this function is used.